pkg/database/bigquery.schema.json

```diff
@@ -82,5 +82,22 @@
     "mode": "REPEATED",
     "name": "nodes",
     "type": "RECORD"
+  },
+  {
+    "fields": [
+      {
+        "mode": "REQUIRED",
+        "name": "name",
+        "type": "STRING"
+      },
+      {
+        "mode": "REQUIRED",
```

Comment:
Is the value REALLY required?  If so, see above comment.  It needs to be validated in code.